game/scripts/vscripts/Sections/Chapter2/SectionCreeps.ts

```diff
@@ -47,9 +52,15 @@ const onStart = (complete: () => void) => {
     const godzSpawnLocation = Vector(-6715, 4402, 128)
     const sniperSpawnLocation = Vector(-5700, 5555, 128)
 
-    const sniperTalkShitLocalizationOptions: LocalizationKey[] = [LocalizationKey.Script_2_Creeps_16, LocalizationKey.Script_2_Creeps_17, LocalizationKey.Script_2_Creeps_18]
+    const sniperTalkShitLocalizationOptions: LocalizationKey[] = [LocalizationKey.Script_2_Creeps_17, LocalizationKey.Script_2_Creeps_18, LocalizationKey.Script_2_Creeps_18_1]
     const chosenSniperTalkShitLocalizationOption = sniperTalkShitLocalizationOptions[RandomInt(0, sniperTalkShitLocalizationOptions.length - 1)]
 
+    const sniperDiesLocalizationOptions: LocalizationKey[] = [LocalizationKey.Script_2_Creeps_24, LocalizationKey.Script_2_Creeps_25]
+    const chosenSniperDiesLocalizationOption = sniperDiesLocalizationOptions[RandomInt(0, sniperDiesLocalizationOptions.length - 1)]
```

Comment:
You can use `randomChoice(array)` from util here and in other places instead of doing it yourself with RandomInt.

Reply:
Didn't know we have this, definitely better. Done!